src/functionaltests/resources/features/restEndpoints.feature

```diff
@@ -21,7 +21,7 @@ Feature: Test Rest Endpoints
     | 200          | /templates/events                      |
     | 200          | /templates/rules                       |
     | 200          | /templates/subscriptions               |
-    | 404          | /queryMissedNotifications/subs_name    |
+    | 404          | /failed-notifications/subs_name        |

```

Comment:
Maybe remove the name here and call just failed-notification, all 404 tests here just dont test anything.

Reply:
I'm not sure why it was even there.

Reply:
Removed these 404 tests since they don't seem to add any value.